packages/tupaia-web-server/src/routes/ProjectCountryAccessListRoute.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate';
+import { TupaiaWebProjectCountryAccessListRequest } from '@tupaia/types';
+
+export type ProjectCountryAccessListRequest = Request<
+  TupaiaWebProjectCountryAccessListRequest.Params,
+  TupaiaWebProjectCountryAccessListRequest.ResBody,
+  TupaiaWebProjectCountryAccessListRequest.ReqBody,
+  TupaiaWebProjectCountryAccessListRequest.ReqQuery
+>;
+
+export class ProjectCountryAccessListRoute extends Route<ProjectCountryAccessListRequest> {
+  public async buildResponse() {
+    const { ctx, params } = this.req;
+    const { projectCode } = params;
+    const { projects } = await ctx.services.webConfig.fetchProjects();
```

Comment:
I don't know if this would work, because the entity server checks permissions right? And we just want to know what countries a project has inside it. The web config server does some useful things with grabbing the country names from the entity hierarchy, and disregards the permissions which I think here is what we want. Eventually that logic can probably move to the central server or something, because it does get used a lot, and I reckon that can probably be done as part of the maintenance ticket for removing the use of web-config-server

Reply:
I did think of that, but I don't think it's been merged in yet :(